ember-app/app/models/new/column.js

```diff
@@ -6,7 +6,37 @@ var Column = Ember.Object.extend(Ember.PromiseProxyMixin, {
   isLast: false,
   filteredContent: Ember.computed.filter('content.@each.columnIndex', function(item){
     return item.get('columnIndex') === this.get('data.index');
-  })
+  }),
+  columns: Ember.computed.alias("board.columns"),
+  isLastColumn: function(){
+    return this.get("columns.lastObject.data.name") === this.get("data.name");
+  }.property("columns.lastObject"),
+  isFirstColumn: function(){
+    return this.get("columns.firstObject.data.name") === this.get("data.name");
+  }.property("columns.firstObject"),
+  sortedIssues: function(){
+    var issues = this.get("board.issues")
+      .filter(this.filterStrategy.bind(this))
+      .sort(this.sortStrategy);
+    return issues;
+  }.property("board.issues.@each.{columnIndex,order,state}"),
+  filterStrategy: function(issue){
+    var issue_index = issue.get("columnIndex");
```

Comment:
To make sure I'm reading this right, this is the only actual logic change within the relocated column functions (was `var issue_index = issue.data.current_state.index`). Everything else is just adjustment to add/remove `model`?